app/models/order.rb

```diff
@@ -8,4 +8,8 @@ class Order <ApplicationRecord
   def grandtotal
     item_orders.sum('price * quantity')
   end
+
+  def status
```

Comment:
The status is good here but I was thinking it would have been much better to add a `status` column in the `item_orders` table with an initial value of `pending` or an enum with an initial of `0`. The order will be pending by default when an order is created.

Reply:
That sounds good to me. I didnt know exactly how to approach it. But that makes sense

Reply:
We can get in a call and try to set it up it will save us a lot in future stories.

Reply:
Sounds good to me. I can get on any time now